dev-tools/reconfigurator-cli/tests/input/cmds-target-release.txt

```diff
@@ -413,5 +413,50 @@ blueprint-diff latest
 # image source set to an artifact from our TUF repo.
 blueprint-show latest
 
-# We ought to update the inventory for the final sled and then step through
-# the Nexus handoff process, but that work is still in progress.
+# Start the Nexus handoff process: A planning step here should place three new
+# Nexus zones, each running from the new TUF repo artifact.
+blueprint-plan latest latest
+blueprint-diff latest
+
+# Update inventory to reflect the three Nexus zones starting.
+sled-set 2b8f0cb3-0295-4b3c-bc58-4fe88b57112c omicron-config latest
+sled-set 98e6b7c2-2efa-41ca-b20a-0a4d61102fe6 omicron-config latest
+sled-set d81c6a84-79b8-4958-ae41-ea46c9b19763 omicron-config latest
+inventory-generate
+
+# Planning now should bump the top-level `nexus_generation` to 2, indicating
+# that we want handoff to begin.
+blueprint-plan latest latest
+blueprint-diff latest
+
+# Planning again should make no changes: we haven't yet performed a handoff,
+# so the old Nexus zones are still in charge.
+blueprint-plan latest latest
+blueprint-diff latest
+
+# Bump the set of active Nexus zones; this simulates Nexus handoff.
+set active-nexus-gen 2
+
+# Planning should now expunge one of the old install dataset Nexus zones.
+blueprint-plan latest latest
+blueprint-diff latest
+
+# Two more planning iterations should expunge the remaining two old install
+# dataset Nexus zones.
```

Comment:
Nitpick, to prevent comment code-rot - yes, these are "install dataset" zones now, but when we do image resolution (presumably as a prerequisite of updating anything) these are just going to be "old zones", and not from the install dataset.

Should we just call them "old zone images" instead of specifying whether they're on the install dataset?